content/terraform-docs-common/docs/cloud-docs/api-docs/private-registry/stack-component-configurations.mdx

```diff
@@ -0,0 +1,689 @@
+| `data.attributes.vcs-repo.github-app-installation-id` | string |      | The VCS Connection GitHub App Installation to use. Find this ID on the account settings page. Requires previously authorizing the GitHub App and generating a user-to-server token. Manage the token from **Account Settings** within HCP Terraform. You can not specify this value if `oauth-token-id` is specified. |
+| `data.attributes.vcs-repo.display_identifier`     | string  |         | The display identifier for the repository. For most VCS providers outside of Bitbucket Cloud, this identifier matches the `data.attributes.vcs-repo.identifier` string.                                                                                       |
+| `data.attributes.vcs-repo.source-directory`       | string  | (optional) | The directory in the repository where the component configuration is located. Useful for monorepo setups where multiple component configurations exist in different directories.                                                                            |
+| `data.attributes.vcs-repo.tag-prefix`             | string  | (optional) | A prefix to filter tags for versioning. Only tags matching this prefix will be considered as versions. Useful for monorepo setups where multiple component configurations share the same repository but use different tag prefixes.                          |
+
+A VCS repository identifier is a reference to a VCS repository in the format `:org/:repo`, where `:org` and `:repo` refer to the organization, or project key for Bitbucket Data Center, and repository in your VCS provider. The format for Azure DevOps is `:org/:project/_git/:repo`.
+
+The OAuth Token ID identifies the VCS connection, and therefore the organization, that the component configuration will be created in.
+
+**For API-only component configurations:**
+
+| Key path              | Type   | Default | Description                                                                                                                        |
+| --------------------- | ------ | ------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| `data.type`           | string |         | Must be `"registry-components"`.                                                                                                    |
+| `data.attributes.name`| string |         | The name of this component configuration. May contain alphanumeric characters, with dashes and underscores allowed in non-leading or trailing positions. Maximum length is 64 characters. |
+
+### Sample Payload (VCS-backed)
+
+**Basic VCS configuration:**
```

Comment:
Can you make these full sentences and remove the bold please? For this and all the other instances?